### DISCUSSION TITLE
How to give badges without using point

### DISCUSSION THREAD

**Discussion by jonasbrant** [exoplatform/exo-community#57] [Q&A]
We would like to use the badge funcion to recognize competencies developed by community members. This way we would like to use the structure to give a badge without use points. only the manager recognition

**Comment by srenault-meeds** [comment]
Hi @jonasbrant this is interesting.

It is currently not possible but we can try to imagine together something about it.

Here is a suggestion:
1. When creating an action so users can contribute
2. Then you'll have to choose how to recognize the contributions among different choices:
- points
- badge
- meeds token

Of course, at least one choice needs to be selected and it must be possible to select multiple choices.
WDYT?

**Reply by jonasbrant** [reply]
This is a great idea. Some times recognition with points, meeds could be ephemerus and a badge could give this long term recognition to the community.
